pbdf/Issues/idin/description.xml

```diff
@@ -25,24 +25,24 @@
 	<BackgroundGradientEnd>#9DDCE5</BackgroundGradientEnd>
 	<IsInCredentialStore>true</IsInCredentialStore>
 	<Category>
-		<en>Personal data</en>
-		<nl>Persoonsgegevens</nl>
+		<en>Personal</en>
+		<nl>Persoonlijk</nl>
 	</Category>
 	<FAQIntro>
-		<en></en>
-		<nl>Zet je persoonsgegevens uit iDIN in je IRMA-app.</nl>
+		<en>Add your personal data from your bank to your IRMA app.</en>
+		<nl>Laad je persoonsgegevens van je bank in je IRMA-app.</nl>
     </FAQIntro>
 	<FAQPurpose>
-		<en></en>
-		<nl>Met jouw persoonsgegevens uit iDIN in de IRMA-app kun jij bewijzen hoe je heet, hoe oud je bent en waar je woont.</nl>
+		<en>With your personal data from your bank in your IRMA app, you can, for instance, prove what your name is, what your date of birth is, and where you live.</en>
+		<nl>Met jouw persoonsgegevens van je bank in de IRMA-app kun jij bewijzen hoe je heet, wat je geboortedatum is en waar je woont.</nl>
 	</FAQPurpose>
 	<FAQContent>
-		<en></en>
-		<nl>Je haalt de volgende gegevens op:\n• initialen\n• achternaam\n• geslacht\n• geboortedatum\n• leeftijd (ouder dan 12, 16, 18, 21 of 65)\n• adres\n• postcode\n• plaats</nl>
+		<en>You will add the following data.\n• initials\n• last name\n• gender\n• date of birth\n• age limits (older than 12, 16, 18, 21 or 65)\n• address\n• postal code\n• city.</en>
+		<nl>Je laadt de volgende gegevens.\n• initialen\n• achternaam\n• geslacht\n• geboortedatum\n• leeftijdsgrenzen (ouder dan 12, 16, 18, 21 of 65)\n• adres\n• postcode\n• woonplaats</nl>
 	</FAQContent>
 	<FAQHowto>
-		<en></en>
-		<nl>Als je een Nederlandse bankrekening hebt, kun je je naam, adres en geboortedatum toevoegen via iDIN (een dienst van de banken).\n\nDoorloop de volgende stappen:\n1. Log in bij je eigen bank\n2. Je ziet de gegevens die je van jouw bank naar je IRMA-app overzet.\n3. Bevestig het overzetten van je gegevens naar je IRMA-app.\n\nDe Privacy by Design Foundation ontvangt je bankgegevens van iDIN en geeft deze door aan je IRMA-app.\n\nJe ontvangt een kaartje met je persoonsgegevens uit iDIN.</nl>
+		<en>If you have a Dutch bank account, you can personal data from the bank to your IRMA app.\n\n To do so, complete the following steps.\n1. Log in to your own bank via the bank's login service called iDIN.\n2. You see the personal data that you can load into your IRMA app.\n3. Confirm that you wish to do so.\nThe Privacy by Design foundation receives your personal data from iDIN and passes them on to your IRMA app.\n\nYou now receive a card with your personal details from your bank.</en>
```

Comment:
`you can personal data`
`Foundation` (capitalized)

Reply:
@bajacobs I will add "add" to resolve the first issue and capitalise Foundation. Ok?

Reply:
Indeed, add "add" but don't capitalise, see previous comment.